packages/strapi-parts/src/Link/Link.stories.mdx

```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
 
 # Link
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
+- Clicking on a Link opens a new tab.
```

Comment:
This is not always true. The "go back button" is a Link and it does not open a new tab 😊

Reply:
Well this is a mistake coming from the design side then. In an ideal world, this should be like mentioned in the documentation

Reply:
I'm not sure to understand what you mean